templates/whatson/whats_on_page.html

```diff
@@ -5,12 +5,71 @@
   {% meta_tags %}
 {% endblock %}
 
+{# Breadcrumb is included inside the header area #}
+{% block breadcrumb %}
+{% endblock %}
+
 {% block content %}
 
+  <div class="full-width-header full-width-header--pink">
+    {% include 'includes/breadcrumb.html' with classes='breadcrumb-content--reduce-space' %}
+
+    {% include "includes/generic-intro.html" with title=page.title intro=page.intro %}
+
+    <div class="tna-container">
+      <div class="tna-column tna-column--full">
+        <form class="filters" method="GET">
```

Comment:
It would be good to specify the `action` parameter (which would be the current URL) here else any URL fragments such as `#maincontent` (if the skip link is used, for example) will persist after the form is posted and the focus will be there rather than at the top of the page (from what I can recall - you may need to test this).

Reply:
Ah thanks - I assumed that detail would come when the back-end is hooked in - I added the GET parameter as Leanne mentioned a need to link straight to filtered lists so I was thinking ahead with that. But it makes sense that the action would be the current URL, so I'll add that in.

Reply:
I've set the action to `#` for now - down the line that might become a specific anchor point on the page so the user lands at the start of the listing.